urldownloader.py

```diff
@@ -179,22 +179,26 @@ def execute(self, request: ServiceRequest) -> None:
             with tempfile.NamedTemporaryFile(dir=self.working_directory, delete=False, mode="w") as temp_conf:
                 yaml.dump(kangooroo_config, temp_conf)
 
+            env_variables = {
+                "JAVA_OPTS": f"-Xmx{math.floor(self.service_attributes.docker_config.ram_mb*0.75)}m"
+            }
+
             kangooroo_args = [
-                "java",
-                f"-Xmx{math.floor(self.service_attributes.docker_config.ram_mb*0.75)}m",
-                "-Dlogback.configurationFile=./logback.xml",
-                "-jar",
-                "KangoorooStandalone.jar",
+                "./bin/kangooroo",
                 "--conf-file",
                 temp_conf.name,
-                "--dont-use-captcha",  # Don't use captcha for the moment, to be enabled later
+                "-mods",
+                "summary,captcha",
+                "--simple-result",
                 "--url",
-                request.task.fileinfo.uri_info.uri,
+                request.task.fileinfo.uri_info.uri
             ]
             if self.no_sandbox:
                 kangooroo_args.insert(-2, "--no-sandbox")
             try:
-                subprocess.run(kangooroo_args, cwd=KANGOOROO_FOLDER, capture_output=True, timeout=self.request_timeout)
+                subprocess.run(kangooroo_args, cwd=KANGOOROO_FOLDER,
```

Comment:
Can you use the settings from .vscode/settings.json so that it applies the right formatter?